Add multiple channel policy properties for the number of channels, capacity and ratio

mariaa144 opened the pull request.

accumulator commented on charge_lnd/policy.py:
```diff
@@ -159,6 +167,78 @@ def match_by_node(self, channel, config):
             return False
 
+        # Consider multiple channels per node policies
+        if set(map(lambda n: "node." + n in config, multiple_chans_props)).issuperset({True}):
```
This could be a bit more legible using
`if any(set(map(lambda n: "node." + n in config, multiple_chans_props))):`

mariaa144 replied:
That is better. Thank you. pylint suggested what I originally had.

accumulator commented:
Very nice additions @mariaa144. However, it is a bit confusing that these new criteria are not about the node as a whole, but specifically about the subset of channels a node has with our node.

For example the property `node.min_channels_active` could be renamed to `node.min_direct_channels_active` and the README could mention that ('match on number of active channels the peer node has **with us**')

mariaa144 commented:
Do you think `node.min_shared_channels_active` might be more clear?

accumulator commented:
Excellent

mariaa144 commented:
- I updated the README so it is clear what the new properties are for
- Add the `shared` test for all the added properties
- Added an example config for how the properties might be used

accumulator commented:
Thanks, great addition. Merged.
